velox/docs/develop/operators.rst

```diff
@@ -167,6 +167,10 @@ individual measures are sorted and de-duplicated.
      - One or more measures to compute. Each measure specifies an expression, e.g. count(1), sum(a), avg(b), optional boolean input column that's used to mask out rows for this particular measure, optional list of input columns to sort by before computing the measure, an optional flag to indicate that inputs must be deduplicated before computing the measure. Expressions must be in the form of aggregate function calls over input columns directly, e.g. sum(c) is ok, but sum(c + d) is not.
    * - ignoreNullKeys
      - A boolean flag indicating whether the aggregation should drop rows with nulls in any of the grouping keys. Used to avoid unnecessary processing for an aggregation followed by an inner join on the grouping keys.
+   * - globalGroupingSets
+     - If the AggregationNode is over a GroupIdNode, then its possible that it computes a global aggregate. The global aggregate is from a GroupId group that has all NULL keys, except for the GroupIdKey. Such a group is listed in the globalGroupingSets. The Aggregation operator constructs a single row with the default value of the aggregate per globalGroupingSet if there is no input.
+   * - groupIdColumn
+     - The output column corresponding to the GroupIdKey (if the AggregationNode is over a GroupIdNode). The groupIdColumn is populated with the global GroupingSet number when there is no input row. The groupIdColumn must be of BIGINT type.
```

Comment:
I feel this explanation is too complex to be understood. Let's add a detailed example of the use case to allow readers to undrerstand better.

groupId should be described as the grouping key that contains the group ID generated by GroupId operator.